### DISCUSSION TITLE
是否可以把全部的挂载盘在转变为smb共享?

### DISCUSSION THREAD

**Discussion by crarm** [AlistGo/alist#2299] [Ideas]
对技术不是太懂啊，不知道能否把alist添加一个smb共享呢，这样多一种方式访问，emby等就可以直接访问网盘内容而不用rclone来转换挂载了。不知道技术上是否可行呢？

**Comment by ChitaGideon** [comment]
个人认为，smb的确比webdav 通用些

**Comment by SeanHeuc** [comment]
alist目前优势在于轻量化只支持webdav，支持的网盘多。你需要的smb用rclone吧，可以使用rclone挂载webdav的方式，这样webdav就转换成了本地挂载。我自己测试过，不是很好用，因为smb更多适用于局域网内高速访问，用任何刮削器都会很卡，需要下载很多数据，而且aliyun还有访问频率限制以至于刮削没法用。

**Reply by Skywind5487** [reply]
有没有中继的软体可以在win11上用?
目前适用win11 上用rclone挂载了alist 的webdav
能不能有什么软体可以再win11上面转换协议，让苹果的档案伺服器能够以smb读到alist的东西?

**Reply by pongfcnkl** [reply]
@Skywind5487 
套娃呗，用rclone本地挂载alist的webdav变成硬盘，然后再把这个硬盘分享出去
所以为什么不直接用webdav？

**Reply by Skywind5487** [reply]
因为苹果的「档案」里面的档案伺服器功能 只支援smb......
很多老机器也是的
@pongfcnkl
